web/ui/react-app/src/Navbar.tsx

```diff
@@ -14,6 +15,14 @@ import {
 } from 'reactstrap';
 import PathPrefixProps from './types/PathPrefixProps';
 
+// Declared/defined in public/index.html, value replaced by Prometheus when serving bundle.
```

Comment:
@juliusv Can you please suggest a better way than this

Reply:
I'm not sure we can do better, but I would prefer if we keep this super magic placeholder stuff to `index.tsx` (where the existing path prefix magic is as well), and then hand it down to the App and the Navbar using a prop.